src/lib/integration_test_cloud_engine/kubernetes_network.ml

```diff
@@ -96,6 +109,17 @@ module Node = struct
     Integration_test_lib.Util.run_cmd_or_error cwd "kubectl"
       (base_kube_args config @ [ "cp"; "-c"; container_id; tmp_file; dest_file ])
 
+  (* Downloads file using cat. Method useful in case when pod does not have tar installed thus cp command is not supported *)
+  let download_file_safe node ~source_file ~target_file =
+    let open Malleable_error.Let_syntax in
+    let%bind content = run_in_pod node ~cmd:[ "cat"; source_file ] in
```

Comment:
Yes but cp need special package installed. Above code bypass this requirement

Reply:
If the necessary package is `tar`, that's already a dependency of our daemon. It's actually a bug if we don't have `tar` installed on those images, since the daemon uses `tar` to bundle up crash reports.

Reply:
Ok, will make a note of that.